source/php/Component/Icon/Icon.php

```diff
@@ -35,19 +47,18 @@ public function init()
         $this->appendSpace();
         $this->setSize();
 
-        //Set material icon class
-        $this->data['classList'][] = "material-icons";
-
         //Do not translate the icon
         $this->data['attributeList']['translate'] = "no";
 
         //Identify as an image
         $this->data['attributeList']['role'] = "img";
         $this->data['attributeList']['aria-label'] = $this->getAltText($icon);
         $this->data['attributeList']['alt'] = $this->getAltText($icon);
+    }
 
-        //Add modifier
-        $this->data['classList'][] = $this->createIconModifier($icon);
+    private function iconIsSvg($icon)
+    {
+        return str_ends_with($icon, '.svg') !== false;
```

Comment:
This is a PHP 8 feature. Backwards compability with 7.4 must be enshured.

Reply:
Closing this, it was decided that we'll use `symfony/polyfill-php80` in Municipio as it's already being used in Modularity.